internal/cmd/sqlserverflex/instance/instance.go

```diff
@@ -1,7 +1,9 @@
 package instance
 
 import (
+	"github.com/stackitcloud/stackit-cli/internal/cmd/argus/credentials/create"
```

Comment:
Wrong import here 👀

Reply:
Great catch, I had to remove the command that I was using to etst before making the PR for the `list` and didn't notice in the re-import